Src/BootCamp.Chapter/Conversions.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace BootCamp.Chapter
+{
+    class Conversions
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Convert Decimal To Binary");
+            ConvertDecimalToBinary();
+            Console.WriteLine();
+            Console.WriteLine("Convert Binary To Decimal");
+            ConvertBinaryToDecimal();
+        }
+
+        private static void ConvertDecimalToBinary()
+        {
+            int i;
+            int[] binaryHolder = new int[40];
+            int[] a = new int[40];
+            Console.Write("Enter the number to convert: ");
+            if (!int.TryParse(Console.ReadLine(), out int decimalNumber))
+            {
+                Console.WriteLine("That wasn't a decimal number. Please try again");
+                ConvertDecimalToBinary();
+            }
+            i = 0;
+            while (decimalNumber > 0)
+            {
+                binaryHolder[i] = decimalNumber % 2;
+                decimalNumber = decimalNumber / 2;
+                i++;
+                for (i = 0; decimalNumber > 0; i++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This loop's stop condition tests 'decimalNumber' but the incrementer updates 'i'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011723)